addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -671,11 +671,27 @@
         </we-row>
         <!-- LINKS -->
         <we-row string="Links" class="o_design_tab_title">
+            <we-input data-customize-css-property=""
+                        data-css-property="font-size"
+                        data-selector-text="a:not(.btn), a.btn.btn-link"
+                        data-unit="px"/>
             <we-colorpicker data-customize-css-property=""
                             data-css-property="color"
                             data-selector-text="a:not(.btn), a.btn.btn-link"
                             data-no-transparency="true"
                             data-color-prefix="text-"/>
+        </we-row>
+        <we-row>
```

Comment:
Just to make sure, did you check there's no issue arising from the fact that not all options of links are in the same `we-row` (eg, was there nothing targeting all the options using a selector like `we-row[string=Links]`)? Are you sure you need to split it into two rows?

EDIT: I see primary button options are already split in several rows but then with the `o_we_sublevel_1` class. Was there no need to use that here to make the grouping clear?